### DISCUSSION TITLE
Can't upload files to the PACS server - Search for Studies failed

### DISCUSSION THREAD

**Discussion by Gunggl** [dcm4che/dcm4chee-arc-light#3857] [Q&A] [answered]
I installed the slim viewer from Hermann lab (https://github.com/herrmannlab/slim) which utilizes a dcm4chee-arc-light DICOMweb server on localhost for local app development and testing on a debian vm.

When I start the viewer, I get an http error 500 on GET http://localhost:8008/dcm4chee-arc/aets/DCM4CHEE/rs/studies?ModalitiesInStudy=SM

Does anybody has an idea what's wrong?

![image](https://user-images.githubusercontent.com/97106150/200022043-01db5163-919b-477e-813f-3effa73d0bc7.png)

[server.log](https://github.com/dcm4che/dcm4chee-arc-light/files/9940133/server.log)

**Comment by vrindanayak** [comment, marked as answer]
> 2022-10-31 12:52:02,491 ERROR [org.dcm4chee.arc.impl.ArchiveDeviceProducer] (ServerService Thread Pool -- 83) Missing Configuration for Device 'dcm4chee-arc' - you may change the Device name by System Property 'dcm4chee-arc.DeviceName'

Ensure [LDAP configuration for the archive is correctly initialized](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Installation#import-default-configuration-into-ldap-server). 
See [similar posts in dcm4che user forum](https://groups.google.com/g/dcm4che/c/ejI0VMNOOsA/m/f8aoH42IBAAJ)
